docs/source/tutorials/index.rst

```diff
@@ -5,6 +5,7 @@ Tutorials
    :maxdepth: 1
 
    datalab/
+   improving_ml_performance
```

Comment:
this is the correct place for the new tutorial, so make the other index.rst file match this location

Reply:
Just fixed the other index.rst file to match this in latest commit